src/test/regress/bin/normalize.sed

```diff
@@ -65,6 +65,14 @@ s/"(ref_table_[0-9]_|ref_table_[0-9]_value_fkey_)[0-9]+"/"\1xxxxxxx"/g
 /^LINE [0-9]+:.*$/d
 /^ *\^$/d
 
+# PG 11 and PG 12 differs in terms of CTE inlining
+# so shadow anything related to subplan ids
+s/generating subplan [0-9]+\_[0-9]+/generating subplan XXX\_XX/g
```

Comment:
Based on the test output, I think the second number is quite stable. Which makes it probably interesting if it changes. So I think this would be better normalization rule:
```suggestion
s/generating subplan [0-9]+\_/generating subplan XXX\_/g
```

Same for this rule below:
```
s/Subplan [0-9]+\_[0-9]+/Subplan XX\_XX/g 
```

Reply:
Actually more of these rules match more than we want. For instance removing InitPlan numbers explain plans, because it matches `Plan [0-9]+]`. So I suggest this change to the normalize script instead.
```sed
# Needed because PG12 and PG11 generated different output, can probably be
# removed when we don't support PG11 anymore, so PG_VERSION_NUM < 120000
s/for CTE/for subquery/g

# Plan numbers are not very stable, subplan numbers are so we keep those
s/DEBUG:  Plan [0-9]+/DEBUG:  Plan XXX/g
s/generating subplan [0-9]+\_/generating subplan XXX\_/g
s/read_intermediate_result\('[0-9]+_/read_intermediate_result('XXX_/g
s/Subplan [0-9]+\_/Subplan XXX\_/g
```